rest/service.go

```diff
@@ -71,12 +73,21 @@ func (s *Service) Validate() error {
 			GetCreateUser: model.GetOrAddUser,
 		})
 		if err != nil {
-			return errors.Wrap(err, "problem setting up user manager")
+			return errors.Wrap(err, "problem setting up ldap user manager")
+		}
+	} else if s.Conf.NaiveAuth.AppAuth {
+		s.um, err = model.NewNaiveUserManager(&s.Conf.NaiveAuth)
+		if err != nil {
+			return errors.Wrap(err, "problem setting up naive user manager")
 		}
 	}
 
-	s.depot, err = depot.NewFileDepot(s.CertPath)
-	grip.Warning(errors.Wrap(err, "no certificate depot constructed"))
+	if s.Depot == nil {
+		grip.Warning(errors.Wrap(err, "no certificate depot provided"))
+		if err != nil {
+			return errors.Wrap(err, "problem setting up naive user manager")
+		}
+	}
```

Comment:
where does the error on line 86 come from? is this actually the naive user manager? 

it seems like we should also continue to allow this setup to run if there isn't a depot?

Reply:
sorry that was a copy and paste error, removing it